src/Configuration/RectorConfigBuilder.php

```diff
@@ -415,7 +415,11 @@ function (string $string): string {
             ->depth(0)
             ->ignoreDotFiles(false)
             ->name('*.php')
-            ->name('.*.php');
+            ->name('.*.php')
+            // this file cannot be interpreted as PHP file
+            // https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html#expected-arguments
+            ->notName('.phpstorm.meta.php')
+        ;
```

Comment:
I think we should not need to enable dot files by default, eg:

```php
public function withRootFiles(bool $includeDotFiles = false)
```

ensure be safe by default, while still hardcoded this obvious `.phpstorm.meta.php` so no change by default.

Then you can add condition:

```php
if ($includeDotFiles) {
     $rootPhpFilesFinder->ignoreDotFiles(false);
}

Reply:
I'm OK with all options (enable for all dotfiles except known risky, enable for known dotfiles, provide an argument, etc)

Reply:
I am for:

- disable by default
- still exlude `.phpstorm.meta.php` as known risky

Reply:
@TomasVotruba, what do you think about this?